cmd/config/internal/commands/tree.go

```diff
@@ -90,7 +85,7 @@ func (r *TreeRunner) runE(c *cobra.Command, args []string) error {
 		input = &kio.ByteReader{Reader: c.InOrStdin()}
 	} else {
 		root = filepath.Clean(args[0])
-		input = kio.LocalPackageReader{PackagePath: args[0], MatchFilesGlob: r.getMatchFilesGlob()}
```

Comment:
How will this affect the output of kustomize cfg tree? Is this what we want for kustomize as well as kpt?

Reply:
Yes. Essentially users would be interested to see resource files summary.